package.json

```diff
@@ -89,7 +92,7 @@
     "type-coverage": "^1.8.0"
   },
   "scripts": {
-    "build": "react-scripts build",
+    "build": "INLINE_RUNTIME_CHUNK=false react-scripts build",
```

Comment:
This var is needed to disable a chunk of js that CRA injects by default. From their [docs](https://facebook.github.io/create-react-app/docs/advanced-configuration):

> By default, Create React App will embed the runtime script into index.html during the production build. When set to false, the script will not be embedded and will be imported as usual. This is normally required when dealing with CSP.

Reply:
Can we move this variable into `.env` instead? (with a comment ideally)

Reply:
Ah good idea 👍